converters/fromarray/pointer.ts

```diff
@@ -0,0 +1,22 @@
+let pointerConverters = [];
+pointerConverters[PointerEventType.Pointer] = pointerFromArray;
+
+export default pointerConverters;
+
+function pointerFromArray(pointerState: any[]): IPointerEventData {
```

Comment:
>IPointerEventData [](start = 48, length = 17)

Can we optimize pointer to only send updates? At least event and position only changes are common.